Camera/schemaDTDL.json

```diff
@@ -44,7 +44,7 @@
             "displayName": "cameraUsage",
```

Comment:
this file is generated automatically. You do not need it on the PR (we try you to work the less the better)

Reply:
done